suzieq/engines/pandas/device.py

```diff
@@ -117,9 +117,10 @@ def get(self, **kwargs):
             df = df.loc[df.status.isin(status)]
         if os_version:
             opdict = {'>': operator.gt, '<': operator.lt, '>=': operator.ge,
-                      '<=': operator.le, '=': operator.eq, '!=': operator.ne}
+                      '<=': operator.le, '=': operator.eq, '!': operator.ne}
             op = operator.eq
             for osv in os_version:
+                osv = osv.replace('!=', '!')
```

Comment:
Can you please add a comment to specify that we do this replacement for backward compatibility?

Reply:
Will do.